includes/class-amp-theme-support.php

```diff
@@ -845,9 +845,8 @@ public static function get_supportable_templates( $options = [] ) {
 	 */
 	public static function add_hooks() {
 
-		// Remove core actions which are invalid AMP.
-		remove_action( 'wp_head', 'wp_post_preview_js', 1 ); // @todo Instead of function, the script output by wp_post_preview_js() should get data-ampdevmode.
-		remove_action( 'wp_head', 'wp_oembed_add_host_js' ); // This is not needed when post embeds are embedded via <amp-wordpress-embed>. See <https://github.com/ampproject/amp-wp/issues/809>.
+		// This is not needed when post embeds are embedded via <amp-wordpress-embed>. See <https://github.com/ampproject/amp-wp/issues/809>.
+		remove_action( 'wp_head', 'wp_oembed_add_host_js' );
```

Comment:
This will be able to be removed in https://github.com/ampproject/amp-wp/pull/6665